src/main/java/org/gridsuite/modification/server/service/NetworkStoreListener.java

```diff
@@ -21,56 +24,98 @@
  */
 public class NetworkStoreListener implements NetworkListener {
 
-    private UUID networkUuid;
+    private UUID groupUuid;
 
     private Network network;
 
     private final NetworkModificationRepository modificationRepository;
 
     private List<ElementaryModificationEntity<?>> modifications = new LinkedList<>();
 
+    private List<CreateEquipmentEntity> creations = new LinkedList<>();
```

Comment:
Is it really necessary to separate the creations
Yhe listener saves all modifications regardless of the type